posts/Yoann/2020-12-11-hibernate-search-6-0-0-Final.adoc

```diff
@@ -0,0 +1,295 @@
+:hsearch-doc-url-prefix: https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/
+:hsearch-migration-url-prefix: https://docs.jboss.org/hibernate/search/6.0/migration/html_single/
+:hsearch-javadoc-url-prefix: https://docs.jboss.org/hibernate/search/6.0/api/
+:hsearch-jira-url-prefix: https://hibernate.atlassian.net/browse
+:hsearch-jira-project-id: 10061
+:hsearch-jira-version-id: 31895
+---
+
+We just published the first stable release of Hibernate Search 6: version 6.0.0.Final.
+
+With more than
+https://hibernate.atlassian.net/issues/?jql=project%20%3D%20HSEARCH%20AND%20fixVersion%20in%20(6.0.0.Final%2C%206.0.0.CR2%2C%206.0.0.CR1%2C%206.0.0.Beta11%2C%206.0.0.Beta10%2C%206.0.0.Beta9%2C%206.0.0.Beta8%2C%206.0.0.Beta7%2C%206.0.0.Beta6%2C%206.0.0.Beta5%2C%206.0.0.Beta4%2C%206.0.0.Beta3%2C%206.0.0.Beta2%2C%206.0.0.Beta1%2C%206.0.0.Alpha9%2C%206.0.0.Alpha8%2C%206.0.0.Alpha7%2C%206.0.0.Alpha6%2C%206.0.0.Alpha5%2C%206.0.0.Alpha4%2C%206.0.0.Alpha3%2C%206.0.0.Alpha2%2C%206.0.0.Alpha1)%20ORDER%20BY%20updated[900 tickets addressed]
+over the course of 20 alphas and betas,
+Hibernate Search 6 really is a major release, and it shows.
+
+The most obvious changes:
+an overhauled API better suited to working with Elasticsearch (but that still works perfectly with embedded Lucene),
+and upgrades from Lucene 5 to 8 and from Elasticsearch 5.6 to 7.
+
+But it doesn't stop there:
+a safer and more concise Search DSL,
+easier mapping,
+more powerful bridges,
+smarter automatic indexing,
+nested documents, ...
+
+== Getting started with Hibernate Search 6
+
+If you want to dive right into the new, shiny Hibernate Search 6,
+a good starting point is the
+link:{hsearch-doc-url-prefix}#getting-started[getting started guide]
+included in the reference documentation.
+
+If you want to migrate an application based on Hibernate Search 5,
+be aware that Hibernate Search 6 APIs differ significantly from Hibernate Search 5.
+We recommend you have a look at
+see link:{hsearch-migration-url-prefix}[the migration guide].
+
+== What's new compared to Hibernate Search 5
+
+=== Dependency upgrades
+
+[[orm-version]]
+==== Hibernate ORM
+
+Hibernate Search 6 relies on Hibernate ORM 5.4.
+
+Note that only Hibernate ORM 5.4.4.Final or later will work correctly;
+5.4.3.Final and earlier will not.
+
+[[lucene-8]]
+==== Lucene 8
+// Old anchor, kept here to avoid dead links
+[[lucene-7]]
+
+The Lucene backend now uses Lucene 8.7.
+
+[[elasticsearch-7]]
+==== Elasticsearch 7
+// Old anchor, kept here to avoid dead links
+[[elasticsearch-6]]
+
+The Elasticsearch backend now works with Elasticsearch 5.6, 6.8 or 7.10.
+Support for older versions of Elasticsearch was dropped.
+
+[[api-refresh]]
+=== API overhaul
+
+This is perhaps the most important aspect of Hibernate Search 6: most APIs have changed.
+We didn't take this decision lightly, and we don't intend to do this again anytime soon:
+at this point, breaking APIs was necessary for Hibernate Search to keep improving.
```

Comment:
In my opinion the point in which we decided to brake the API is in the past, so I wouldn't write `this point`. Maybe we could just remove `at this point`.